src/test/obj_fragmentation2/obj_fragmentation2.c

```diff
@@ -205,7 +205,7 @@ static float workloads_stat_target[] = {
 };
 
 static float workloads_defrag_stat_target[] = {
-	0.01f, 0.01f, 0.01f, 0.02f, 0.02f, 0.04f, 0.08f, 0.12f, 2100.f
+	0.01f, 0.01f, 0.01f, 0.02f, 0.02f, 0.29f, 0.09f, 0.12f, 2100.f
```

Comment:
the first value is unusually high, @pbalcer is this correct?

Reply:
Yea, I expected a minuscule difference - I'll check this myself.